src/forms/components/encounter/ohri-encounter-form.tsx

```diff
@@ -133,15 +133,39 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
         } else {
           field.isHidden = false;
         }
+
+        //evaluate question-level markdown visibility
+        if (field.markdown?.hide) {
+          field.markdown.isHidden = eval(field.markdown.hide);
+        }
+
         return field;
       }),
     );
+
+    //prepare markdown
+    if (form.markdown && form.markdown.hide) {
```

Comment:
@larslemos I thought markdowns are just fields or questions that don post any obs?